modules/ocm-networking-tab.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// ocm/ocm-overview.adoc
+
+[id="ocm-networking-tab_{context}"]
+= Networking tab
+
+The networking tab provides a control plane API endpoint as well as the default application router. Both the control plane API endpoint and the default application router may be made private by selecting the respective box below each of them.
+
+[NOTE]
+====
+For STS installations, these options cannot be changed.
```

Comment:
This tab is not supported for standard GCP clusters without capability "capability.organization.create_gcp_non_ccs_cluster" . Should we clarify it here?

Reply:
STS should _currently_ only show up for ROSA clusters, not even for OSD clusters yet. And ROSA is AWS-only, so it should be unnecessary to call out lack of GCP support

Reply:
This comment is not related to sts. I mean the networking tab is not supported for standard GCP clusters expect the organization with "capability.organization.create_gcp_non_ccs_cluster"